lib/data_stores/mysql_advanced_data_store.py

```diff
@@ -382,11 +400,32 @@ def _BuildInserts(self, values):
 
     return [aff4_q, subjects_q, attributes_q]
 
-  def _ExecuteTransaction(self, transaction):
+  def _ExecuteQuery(self, query, args=None):
```

Comment:
I forgot about the Transaction piece.  As long as it all gets refactored with the new name then I don't see any problem.  Though the super call is probably not a good idea :)